packages/sdk/src/evm/contracts/prebuilt-implementations/pack.ts

```diff
@@ -66,6 +73,7 @@ export class Pack extends StandardErc1155<PackContract> {
 
   public abi: Abi;
   public metadata: ContractMetadata<PackContract, typeof PackContractSchema>;
+  public app: ContractAppURI<BaseContract>;
```

Comment:
Any reason we're passing `BaseContract` into these? We should be passing in the contract specific type for the prebuilts - like in this case `ContractAppURI<PackContract>`

Reply:
sounds good